Modules/IndividualAssessment/classes/Members/class.ilIndividualAssessmentMember.php

```diff
@@ -305,6 +363,17 @@ public function withFinalized() {
 	}
 
 	/**
+	 * Clone this object and set changed.
+	 *
+	 * @return	ilIndividualAssessmentMember
+	 */
+	public function withChanged() {
+		$clone = clone $this;
+		$clone->changed = 1;
+		return $clone;
+	}
```

Comment:
I don't think we need that, since record was not changed if there is no changer. See db_update.